### PR TITLE
Subtask/as 2637 amend e2e tests

### PR DESCRIPTION
**Pull request by shilpabasu22**
## Ticket Number

https://pins-ds.atlassian.net/browse/AS-2637

## Checklist

- [ ] Requires infrastructure changes
- [ ] If adding or remove environment variables (e.g. in `docker-compose.yaml`) then I have updated the appropriate Helm chart
- [ ] I have updated the documentation accordingly
- [x] My commit history in this PR is linear
- [ ] New features have tests
- [ ] Breaking change (team conversation required)

## Important

Please do not merge from `master` (please only [rebase](https://github.com/foundry4/appeal-planning-decision/wiki/An-intro-to-Git-Rebase)). This keeps the history linear and easier to debug.

### CODE REVIEW

File: lpa-submissions-e2e-tests/cypress/integration/accuracyAppellantsSubmission/accuracyAppellantsSubmission.step.js [hunk 1]
```diff
@@ -95,7 +95,7 @@ Then('the user is shown the error message {string}', (errorMessage) => {
 });
 
 Then(`the user remains in the Accuracy of the appellant's submission page`, () => {
-  cy.verifyPageTitle(pageTitle);
+  cy.title().should('include', 'Error:');
```
Comment (a6software) [inline lpa-submissions-e2e-tests/cypress/integration/accuracyAppellantsSubmission/accuracyAppellantsSubmission.step.js:98]: A small potential improvement here might be to use a regex match:

`cy.title().should('match', /^Error: /);`

The benefit to this is that it checks the string starts with 'Error:', whereas the `include` approach would match:

* 'There is an Error: here'
* 'Error: something'
* 'Error:'

And so on. Depends if you need to be that specific.

File: lpa-submissions-e2e-tests/cypress/integration/appealsDevelopmentPlan/appealsDevelopmentPlanSteps.js [hunk 2]
```diff
@@ -95,13 +95,21 @@ Then(`the LPA Planning Officer is presented with the ability to provide informat
...
+        errorMessage,
+        '[data-cy="plan-changes-text-error"]',
+        'plan-changes-text',
+      );
 });
 
 Then(`the LPA Planning Officer remains on the page`, () => {
   cy.verifyPageHeading(pageHeading);
-  cy.verifyPageTitle(pageTitle);
+  cy.title().should('include', 'Error:');
```
Comment (davehlewis) [inline lpa-submissions-e2e-tests/cypress/integration/appealsDevelopmentPlan/appealsDevelopmentPlanSteps.js:112]: As discussed, I recommend we move this into the cy.validateErrorMessage command, that way we don't have to explicitly call it every time.

**Review by davehlewis — CHANGES_REQUESTED** [review]

File: lpa-submissions-e2e-tests/cypress/integration/accuracyAppellantsSubmission/accuracyAppellantsSubmission.step.js (hunk 1 above)
Comment (shilpabasu22) [reply lpa-submissions-e2e-tests/cypress/integration/accuracyAppellantsSubmission/accuracyAppellantsSubmission.step.js]: Updated as per review comments

File: lpa-submissions-e2e-tests/cypress/integration/appealsDevelopmentPlan/appealsDevelopmentPlanSteps.js (hunk 2 above)
Comment (shilpabasu22) [reply lpa-submissions-e2e-tests/cypress/integration/appealsDevelopmentPlan/appealsDevelopmentPlanSteps.js]: Updated as per review comments

**Review by a6software — APPROVED** [review]
LGTM - though I would caveat with me not being the best person to approve t2 approach to e2e 👍

**Review by davehlewis — APPROVED** [review]

**Pull request merged**